tools/goctl/rpc/cli/cli.go

```diff
@@ -81,13 +82,19 @@ func RPCNew(_ *cobra.Command, args []string) error {
 		return err
 	}
 
+	protocPath, err := env.LookUpProtoc()
```

Comment:
It's not need to be checked here, and the rpc environment will be tested uniformly in https://github.com/zeromicro/go-zero/blob/master/tools/goctl/rpc/generator/gen.go#L47